io/net/virt-net/lpm.py

```diff
@@ -165,15 +181,28 @@ def rsct_service_start(self):
         '''
         try:
             for svc in ["rsct", "rsct_rm"]:
-                process.run('startsrc -g %s' % svc, shell=True, sudo=True)
+                cmd = 'startsrc -g %s' % svc
+                if self.using_peer: 
+                    output = self.peer_session.cmd(cmd)
```

Comment:
get rid of the `output = ` you're not using that variable, you're just sending a command